mobileChatExamples/iOSChatExample/AmazonConnectChatIOSDemo/Models/ChatManager.swift

```diff
@@ -178,6 +195,13 @@ class ChatManager: ObservableObject {
             switch result {
             case .success:
                 print("Participant connection successfully created.")
+                self?.fetchTranscript { success in
+                    if success {
+                        print("All transcripts fetched successfully")
+                    } else {
+                        print("Failed to fetch transcripts")
```

Comment:
dont we need to display err message when transcript failed to fetch ? when it will fail to fetch the transcript?